### ISSUE TITLE
How-to?

### ISSUE THREAD

**Issue by SAGV** [theMozcorp/angular2-papa-csv-parser#1]
Hello Andrius Mozuraitis,

Could you provide a short read me with instructions of "how to use" the module?

**Comment by pantonis** [comment]
@SAGV Did you find a way to use it? I am also interested and stuck

**Comment by SAGV** [comment]
@pantonis not exactly. I ended up writing an essay named [Why Angular 2 sucks](https://medium.com/dirtyjs/why-angular-2-4-5-6-sucks-afb36567ad68), moving the whole project to [Vue.js](https://vuejs.org/) framework and using a [bare](http://papaparse.com/) version of papa.

But before that I made it work with Angular. Using bare version of Papa, too :-)

See [this gist](https://gist.github.com/SAGV/eb9d10b555564fa5479bd63a62008451) for an example (but note that I've cut some parts like vars with my api points/tokens).

**Comment by ruffiem** [comment]
@SAGV @pantonis 

Ever heard about typings ? It's not an Angular problem. 

Instead of using the angular2-whatsoever project, you should rely on the original library and install the types for TypeScript. If the typings does not exist, you can create your own which is pretty easy to do. Plus, if the library is done by the rules (modular), the compiler will import only what you need in the build (tree-shaking much).

For **papaparse**:

Install the bare papaparse version and its typings:
```
npm i papaparse --save
npm i @types/papaparse --save
```

To import papaparse, use:
`import * as papa from 'papaparse';`

Then, wherever in your component:
`papa.parse('someFile.csv');`

You don't need modules since it's imported by the compiler (I highly recommend @angular-cli). You just import it and use all its method declared in the typings.

Cheers ^^
